dor-workflow-service.gemspec

```diff
@@ -17,13 +17,13 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(spec)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency "activesupport", '>= 3.2.1', '< 4.2.2'
+  gem.add_dependency "activesupport", '>= 3.2.1', '< 5'
   gem.add_dependency "nokogiri", '~> 1.6.0'
   gem.add_dependency "rest-client", '~> 1.7'
   gem.add_dependency "confstruct", '~> 0.2.7'
 
   gem.add_development_dependency "rake"
-  gem.add_development_dependency "rspec"
+  gem.add_development_dependency "rspec", "~> 3.3"
```

Comment:
ah, ok, thanks.  i was under the mistaken impression that both would allow you to run 1.7 or 1.8 (or any 1.x), and that the pinning disregarded what i'll call the "least significant version field" (since i can't think of a more concise description).  can't remember whether i thought i read that in documentation or observed that during a bundle update, but that'd been my previous understanding of things.